etc/transpile.yml

```diff
@@ -0,0 +1,15 @@
+logging:
```

Comment:
Is there a way to tell Transcrypt to use two or four spaces rather than tabs in the generated JS?